frameworks/Python/aiohttp/nginx-entrypoint.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/sh
+set -e
+CORES=$(nproc)
+echo "$CORES cores detected, starting $CORES aiohttp workers..."
+
+for i in $(seq 0 $((CORES-1))); do
+  SOCKET="/run/aiohttp-$i.sock"
+  echo "Starting worker on socket $SOCKET"
+  python3 -O -m app.app --socket $SOCKET &
+done
+
+echo "Waiting for all workers to be ready..."
+for i in $(seq 0 $((CORES-1))); do
+  SOCKET="/run/aiohttp-$i.sock"
+  until [ -S "$SOCKET" ]; do
+    echo "Waiting for socket $SOCKET..."
+    sleep 0.2
+  done
+  chown root:www-data "$SOCKET"
+  chmod 660 "$SOCKET"
+done
+
+cat > /aiohttp/nginx.conf <<EOF
+user www-data;
+worker_processes auto;
+
+events {
+    worker_connections 65535;
+}
+
+http {
+    keepalive_requests 10000000;
+
+    upstream aiohttp {
+        least_conn;
```

Comment:
Is there any difference in performance if we drop this and use the default round-robin? Guessing this isn't really needed given that requests should be around 1ms.